.gitignore

```diff
@@ -29,7 +29,7 @@ npm-debug.log
 .pnpm-debug.log
 phpcs.xml
 .phpunit.result.cache
-.docker-env.json
+jetpack-docker-config.json
```

Comment:
Should this be like `/tools/docker/jetpack-docker-config.json` and be moved to around line 40 with the other docker stuff?

Reply:
Yeah, that makes sense. My initial idea to have in the root is that it is a dev-environment configuration, and having it close to other env-related configs was the most obvious solution.